src/pages/home/report/ReportActionItem.js

```diff
@@ -267,6 +271,24 @@ function ReportActionItem(props) {
                     isHovered={hovered}
                 />
             );
+        } else if (props.action.actionName === CONST.REPORT.ACTIONS.TYPE.REIMBURSEMENTQUEUED) {
+            const submitterDisplayName = PersonalDetailsUtils.getDisplayNameOrDefault(props.personalDetailsList, [props.report.ownerAccountID, 'displayName'], props.report.ownerEmail);
+            children = (
+                <ReportActionItemReimbursementQueued
+                    submitterDisplayName={submitterDisplayName}
+                    isCurrentUserSubmitter={ReportUtils.isCurrentUserSubmitter(props.report.reportID)}
+                />
+            );
+        } else if (props.action.actionName === CONST.REPORT.ACTIONS.TYPE.REIMBURSED) {
+            const totalAmount = ReportUtils.getMoneyRequestTotal(props.report);
+            const submitterDisplayName = PersonalDetailsUtils.getDisplayNameOrDefault(props.personalDetailsList, [props.report.ownerAccountID, 'displayName'], props.report.ownerEmail);
+            children = (
+                <ReportActionItemReimbursed
+                    amount={CurrencyUtils.convertToDisplayString(totalAmount, props.report.currency)}
+                    submitterDisplayName={submitterDisplayName}
+                    isFromSubmitterAddingBankAccount={props.action.originalMessage.isSubmitterAddingBankAccount}
+                />
+            );
```

Comment:
I wonder if we can just use [ReportActionItemMessage](https://github.com/Expensify/App/blob/27e9eaa99b8c667fdb05bc672c550c5c67f1d55b/src/pages/home/report/ReportActionItemMessage.js). We already use it for a few system messages, e.g [here](https://github.com/Expensify/App/blob/27e9eaa99b8c667fdb05bc672c550c5c67f1d55b/src/pages/home/report/ReportActionItem.js#L296-L301)

Reply:
Hmm maybe not for `ReportActionItemReimbursementQueued` since we have a button

Reply:
I don't think so because we're not using the fragmented message from the reportAction